activestorage/app/models/active_storage/variant_with_record.rb

```diff
@@ -4,6 +4,7 @@
 # ActiveStorage::VariantRecord. This is only used if `ActiveStorage.track_variants` is enabled.
 class ActiveStorage::VariantWithRecord
   attr_reader :blob, :variation
+  delegate :service, to: :blob
```

Comment:
Is this change necessary? 🤔

Reply:
Since the VariantWithRcord was introduced a year ago I was confused why the common behavior was not extracted in some kind of a base class or an interface. Talking about this case, the read_image test helper is expecting that the variant responds to the service method. Is there is any problem with that?

Reply:
@crawler Good catch, if there's a way to fix our tests without changing behavior that would be best.

I know there has been discussions on how to deal with these generated models, but no movement yet afaik.